tests/testthat/test_utils.R

```diff
@@ -91,6 +106,80 @@ test_that("test_writeTables", {
   expect_equal(outData[c(82,167),], expData, tolerance = 1e-05)
 })
 
+test_that("test_writeOutFeatureAlignmentMap",
+          {
+            #### Prepare data and outout ####
+            dataPath <- system.file("extdata", package = "DIAlignR")
+            fileInfo <- getRunNames(dataPath, oswMerged = TRUE)
+
+            multiFeatureAlignmentMap <- list(`1338` = structure(list(reference_feature_id = structure(c(4.75292059933946e+94,
+                                                                                                        5.52612460272647e-11, 3.59272034753732e+234, 3.74370892823184e+230,
+                                                                                                        2.34380646139584e+234, 4.75292059933946e+94, 5.52612460272647e-11,
+                                                                                                        3.59272034753732e+234, 3.74370892823184e+230, 0, 0, 0, 0, 0,
+                                                                                                        0), class = "integer64"), experiment_feature_id = structure(c(2.04161779709829e+87,
```

Comment:
experiment_feature_id should be in the newline

Reply:
done.